src/main/java/org/takes/facets/fork/FkRegex.java

```diff
@@ -65,9 +65,9 @@
  *       &#64;Override
  *       public Response act(final RqRegex req) {
  *         // Here we immediately getting access to the
- *         // matcher that was used during parsing of
+ *         // match that was used during parsing of
  *         // the incoming request
- *         final String file = req.matcher().group(1);
+ *         final String file = req.match().group(1);
```

Comment:
@mkordas A Refactoring mistake. Corrected it. ( i will push the change with other changes)